pkg/middleware/auth.go

```diff
@@ -64,10 +64,15 @@ func writeRedirectCookie(c *models.ReqContext) {
 		redirectTo = setting.AppSubUrl + c.Req.RequestURI
 	}
 
-	// remove any forceLogin=true params
-	redirectTo = removeForceLoginParams(redirectTo)
+	if redirectTo != "/" {
```

Comment:
Nit: If you invert this clause to a guard clause with something like

```suggestion
	if redirectTo == "/" {
		return
	}
```

(not directly applicable, GitHub didn't let me propose the suggestion on the whole block)
we'd keep the main flow left-aligned.

Reply:
Sounds good!